src/Sulu/Bundle/ContactBundle/Entity/ContactRepository.php

```diff
@@ -60,7 +60,7 @@ public function findById($id)
             ->addSelect('mainContact')
             ->addSelect('account')
             ->addSelect('urls')
-            ->addSelect('partial tags.{id,name}')
```

Comment:
we should not built ontop of such features, but I understand that we did try to avoid fetch changer, creator, ... on all entities